src/main/resources/initdata/asdf.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Is this considered committing the Json?

Reply:
Yes it's for packaging data inside the jar